t/29-backend-ikvm.t

```diff
@@ -13,7 +14,9 @@ use OpenQA::Test::TimeLimit '5';
 use backend::ikvm;    # SUT
 
 $bmwqemu::vars{WORKER_HOSTNAME} = 'localhost';
-ok my $backend = backend::ikvm->new(), 'backend can be created';
+$bmwqemu::vars{"NO_DEPRECATE_BACKEND_IKVM"} = 1;
```

Comment:
the only thing that testing the die command would do is that the die command dies. I guess we can trust that this happens :) Because the alternatives are that either the command never dies which we would realize as soon as we actually delete the deprecated backends and then people that we assume don't exist yell at us that they actually use the backend and they have never got the message about the deprecation. Or it dies anyway because of using it incorrectly and then we have an error with stack trace or something which hopefully somebody would also tell us about eventually.
*But* I put in a test for that now anyway :)